extensions/interactions/InteractiveMap/directives/OppiaInteractiveInteractiveMapDirective.js

```diff
@@ -52,6 +54,20 @@ oppia.directive('oppiaInteractiveInteractiveMap', [
               opacity: '0.8'
             };
           };
+          var ICON = {
```

Comment:
Seems like a weird constant name. Maybe just inline it in addNewMarker() since it's only used once? You can precompute the URLs beforehand and store them as top-level variables if you like.

Reply:
I wasn't sure about the name earlier, I wanted to move it to a oppia.constant so that I can use it in other places but it uses UrlInterpolationService so not possible). I've moved this to the dict as suggested.